go/mysql/conn.go

```diff
@@ -787,15 +787,15 @@ func (c *Conn) writeOKPacketWithHeader(packetOk *PacketOK, headerType byte) erro
 	// assuming CapabilityClientProtocol41
 	length += 4 // status_flags + warnings
 
+	hasSessionTrack := c.Capabilities&CapabilityClientSessionTrack == CapabilityClientSessionTrack
```

Comment:
Nit, but using ` != 0` is more obvious and standard — at least within Vitess — although there's no practical change.